components/nexus-core/src/main/java/org/sonatype/nexus/proxy/repository/RepositoryStatusCheckerThread.java

```diff
@@ -53,6 +60,18 @@ public void run() {
     boolean interrupted = false;
 
     while (isRunning() && getRepository().getProxyMode() != null) {
+
+      // wait for Nexus to fully start
+      if (!systemStatus.isNexusStarted()) {
+        try {
+          Thread.sleep(10 * 1000);
```

Comment:
the smaller the better, but your explanation seems to make it a moot point so nm